### ISSUE TITLE
dnf writes evade from biotop

### ISSUE THREAD

**Issue by myllynen** [iovisor/bcc#1476]
On Fedora 27 with / on XFS on LVM running something like "dnf install libreoffice" triggers several hundreds megabytes of downloading activity but while downloading and storing the packages biotop basically never lists dnf in its output. Meanwhile opening a large file in a text editor and saving the file repeatedly (still on XFS/LVM) causes the editor being included in biotop reporting. With dnf cache/lib dirs on an ext4 fs without LVM then dnf is reported as expected.

If this issue will be addressed as part of #826 then please feel free to close this one. Thanks.

**Comment by brendangregg** [comment]
Do you have a reason to think the dnf writes are not async?

**Comment by myllynen** [comment]
No reason, I suspect they are async. I now checked biotop man page and code and they do not mention whether that would be relevant or not? Thanks.

**Comment by mcaleavya** [comment]
Would this not just show as the flush daemon as the data will be coming
from the network and entering the page cache and then flushed every 30
seconds etc.

**Comment by myllynen** [comment]
Yes, I think I saw a kworker process being quite visible in biotop during my quick test but if doing pid filtering that might be missed altogether.

FWIW, the big picture is that I'm experimenting with PCP BCC PMDA plugin [1]:

https://github.com/myllynen/pcp-bcc-pmda

Thanks.

**Comment by brendangregg** [comment]
Yes, I can add notes to the man pages / example files, and direct people to use the file system tools now that they exist (ext4slower, etc) which should always show the right process.
